do-not-remove/projects_guidelines.md

```diff
@@ -0,0 +1,48 @@
+﻿# Guidelines for our GitHub projects
+
+In this place we will have the documentation for how to handle issues
+among projects of **TecnoYucas**.
+
+## General notes
+
+### Some rules first about the project
+
+1. Every repo must have a automatic Kanban-like GitHub project
+2. Every single pull request MUST be associated with a GitHub issue
+3. Whenever you close a pull request or comment on an issue, please
+make sure to agree on issue completion.
+4. All issues should have a deliverable product (a document or a
```

Comment:
This is not consistent with our current `DO NOT REMOVE` issues. Let’s rephrase this to be relative to the Publication-N repos and not the meta repo.

Reply:
@sadasant When I say "deliverable product", I mean, a generic way to give a "tangible" solution for an issue, the issue could be related or not to a publication. For example, this is not a document considered as TecnoYucas publication, therefore, it is just a guideline document, a "deliverable product" of this issue. What I can do is to put a note that states that a "Deliverable product" could also be a Publication-N.

I am guessing TecnoYucas will be wider than just publications. We will be doing things not related to publications. For example: A merged feature, is a "deliverable product" of an issue but as you know, it does not have necessarily be a publication-related issue.

Reply:
Yes! I understand that, I think. However, we have created issues that have no deliverables, which are our DO NOT REMOVE issues. We use them as a place to communicate. These issues were created with the goal of not removing them for now. What are your thoughts around that?